components/footer/footerLink/FooterLink.style.js

```diff
@@ -0,0 +1,18 @@
+import styled from 'styled-components';
+import { colors } from '../../../style/colors';
+import { P } from '../../../style/typography';
+
+const FooterLinkWrapper = styled(P).attrs({
+  as: 'a',
+})`
+  color: ${colors.WHITE};
+`;
+
+const FooterLinkContainer = styled.div`
+  margin: 0.25em;
+  &:hover {
```

Comment:
Do we want to touch base with the designers for what the hover state would be? I'm guessing underline or bold

Reply:
Good pointing sent a message to jay about this